consoles/VNC.pm

```diff
@@ -587,6 +587,10 @@ sub shift_keys () {
 
 ## use critic
 
+sub _invalid_mapping ($self, $key) {
+    die "No map for '$key' - layouts other than en-us are not supported"
```

Comment:
this can just be a free method

```suggestion
sub die_on_invalid_mapping ($key) {
    die "No map for '$key' - layouts other than en-us are not supported"
```

Reply:
I was assuming we'll eventually use the layout in the error message but I don't really care